README.md

```diff
@@ -1 +1,51 @@
-# catalog-of-my-things
\ No newline at end of file
+# Catalog of my things
+
+> This project consists of a console app to keep a record of different types of items: books and music albums. The data will be stored in JSON files.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+### Prerequisites
+- Ruby >= 3.0.0
+
+### Setup
```

Comment:
- [ ] Apart from setup please add a section with instructions on how to use your app.